example/s3select_example.cpp

```diff
@@ -218,8 +218,9 @@ class awsCli_handler {
       header_size = create_header_records(m_buff_header.get());
       m_result.append(m_buff_header.get(), header_size);
       m_result.append(PAYLOAD_LINE);
+      bool limit_reached = false;
       //status = m_s3_csv_object->run_s3select_on_stream(m_result, input, input_length, s->obj_size);
-      status = m_s3_csv_object->run_s3select_on_stream(m_result, input, input_length, object_size);
+      status = m_s3_csv_object->run_s3select_on_stream(m_result, input, input_length, object_size, limit_reached);
```

Comment:
the `run_s3select_on_stream` interface should not be changed per limit operator.
upon reaching the limit the engine should return a status reporting that (done/complete)

that status will be used also by RGW.